tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -31,42 +32,47 @@ ENV GCSFUSE_PATH "$GOPATH/src/$GCSFUSE_REPO"
 RUN go get -d ${GCSFUSE_REPO}
 
 WORKDIR ${GCSFUSE_PATH}
+ARG DEBEMAIL="gcs-fuse-dev@google.com"
+ARG DEBFULLNAME="GCSFuse Team"
+
 # Build Arg for building through a particular branch/commit. By default, it uses
 # the tag corresponding to passed GCSFUSE VERSION
-ARG BRANCH_NAME="v${GCSFUSE_VERSION}"
+#ARG BRANCH_NAME="v${GCSFUSE_VERSION}"
```

Comment:
why is this commented out and how is the next line working?

Reply:
Sorry, I commented it out while testing to checkout my local branch. Reverted the change.